Jenkinsfile

```diff
@@ -0,0 +1,306 @@
+#!/usr/bin/env groovy
+
+library identifier: 'apm@master',
+changelog: false,
+retriever: modernSCM(
+  [$class: 'GitSCMSource', 
+  credentialsId: 'f6c7695a-671e-4f4f-a331-acdce44ff9ba', 
+  remote: 'git@github.com:elastic/apm-pipeline-library.git'])
+
+pipeline {
+  agent any
+  environment {
+    BASE_DIR="src/github.com/elastic/apm-agent-java"
+    JOB_GIT_CREDENTIALS = "f6c7695a-671e-4f4f-a331-acdce44ff9ba"
+  }
+  triggers {
+    cron('0 0 * * 1-5')
+  }
+  options {
+    timeout(time: 1, unit: 'HOURS') 
+    buildDiscarder(logRotator(numToKeepStr: '3', artifactNumToKeepStr: '2', daysToKeepStr: '30'))
+    timestamps()
+    preserveStashes()
+    //see https://issues.jenkins-ci.org/browse/JENKINS-11752, https://issues.jenkins-ci.org/browse/JENKINS-39536, https://issues.jenkins-ci.org/browse/JENKINS-54133 and jenkinsci/ansicolor-plugin#132
+    //ansiColor('xterm')
+    disableResume()
+    durabilityHint('PERFORMANCE_OPTIMIZED')
+  }
+  parameters {
+    string(name: 'branch_specifier', defaultValue: "", description: "the Git branch specifier to build (branchName, tagName, commitId, etc.)")    
+    string(name: 'GO_VERSION', defaultValue: "1.10.3", description: "Go version to use.")
```

Comment:
can be removed?

Reply:
sure, I needed it for the benchmarks step, I have to correct it to do not need this variable.